packages/utils/src/commonwell/doc-query-shared.ts

```diff
@@ -0,0 +1,103 @@
+import { MetriportMedicalApi } from "@metriport/api-sdk";
+import { sleep } from "@metriport/core/util/sleep";
+import axios from "axios";
+
+export type DetailedConfig = {
+  patientChunkDelayJitterMs: number;
+  queryPollDurationMs: number;
+  maxQueryDurationMs: number; // CW has a 70s timeout, so this is the maximum duration any doc query can take
+  maxDocQueryAttemts: number;
+  /**
+   * If the doc query returns less than this, we want to query again to try and get better
+   * coverage. Had situation where we requeried a patient w/ 1 doc ref and it jumped from
+   * 1 to 20.
+   */
+  minDocsToConsiderCompleted: number;
+};
+export type QueryParams = {
+  cxId: string;
+  patientId: string;
+  apiUrl: string;
+  apiKey: string;
+  triggerWHNotificationsToCx: boolean;
+  config?: DetailedConfig;
+  log?: typeof console.log;
+};
+
+export async function queryDocsForPatient({
+  cxId,
+  patientId,
+  apiUrl,
+  apiKey,
+  config = {
+    patientChunkDelayJitterMs: 1000,
```

Comment:
what's all this stuff? doesn't look related

Reply:
They are for TODO 1106 mentioned by @leite08 above.

Reply:
I don't understand... [this file is already on develop](https://github.com/metriport/metriport/blob/develop/packages/utils/src/commonwell/doc-query-shared.ts), why is it showing up as an addition? Didn't you notice that? Is your branch/branches synchronized with `develop` (`git rebase develop`)?